src/site/content/en/blog/css-props-and-vals/index.md

```diff
@@ -0,0 +1,206 @@
+options](https://drafts.css-houdini.org/css-properties-values-api/#supported-names)
+ranging from numbers to colors to
+[`<custom-ident>`](https://developer.mozilla.org/en-US/docs/Web/CSS/custom-ident)
+types. These syntaxes can also be modified by using the following values
+
+* Appending `+` signifies that it accepts a space-separated list of values of
+  that syntax. For example,  `<length>+` would be a space-separated list of
+  lengths
+* Appending`#` signifies that it accepts a comma-separated list of values of
+  that syntax. For example,  `<color>#` would be a comma-separated list of
+  colors
+* Adding `|` between syntaxes or identifiers signifies that any of the provided
+  options are valid. For example, `<color># | <url> | magic` would allow either
+  a comma-separated list of colors, a URL, or the word `magic`.
+
+### Gotchas
+
+There are two gotchas with Houdini Props and Vals. The first is that, once
+defined, there's no way to update an existing registered property, and trying to
+re-register a property will throw an error indicating that it's already been
+defined.
+
+Second, unlike standard properties, registered properties aren't validated when
+they're parsed. Rather they're validated when they're computed. That means both
+that invalid values won't appear as invalid when inspecting the element's
+properties, and including an invalid property after a valid one won't fall back
+to the valid one; an invalid property will, however, fall back to the registered
+property's default.
+
+## Animating custom properties
+
+A registered custom property provides a fun bonus beyond type checking: the
+ability to animate it! A basic animation example looks like this:
+
+```html
+<script>
+CSS.registerProperty({
+  name: '--stop-color',
+  syntax: '<color>',
+  inherits: false,
+  initialValue: 'blue',
+});
+</script>
+
+<style>
+button {
+  --stop-color: red;
+  transition: --stop-color: 1s;
+}
+
+button:hover {
+  --stop-color: green;
+}
+</style>
+```
+
+When you hover over the button, it'll animate from red to green! Without
+registering the property, it'll jump from one color to the other Because,
+without being registered, fhe browser doesnt know what to expect between one
+value and the next and therefore can't guarentee the ability to transition themß.
+This example can be taken a step further, though, to animate CSS gradients! The
+following CSS can be written with the same registered property:
+
+```css
+button {
+  --stop-color: red;
+  background: linear-gradient(var(--stop-color), black);
+  transition: --stop-color: 1s;
+}
+
+button:hover {
+  --stop-color: green;
+}
+```
+
+This will animate our custom property that's part of the `linear-gradient`, thus
+animating our linear gradient. Check out the Glitch below to see the full code
+in action and play around with it yourself.
+
+<!-- Copy and Paste Me -->
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
```

Comment:
btw the third panel in the glitch look like it's had some of its vertical content cut off for me:

![image](https://user-images.githubusercontent.com/1066253/65088054-c23d3f80-d96c-11e9-8c2f-a9f0ffbd247f.png)

Reply:
@Snugug FYI